pages/docs/tooling/tooling-aste.md

```diff
@@ -0,0 +1,141 @@
+| -t   | Tag for the data created (default="MyScalar")                                       |
+| -it  | The tag for input data. Used in diff mode. If not given tag is used.                |
+| -l   | Logging level (default="INFO")                                                      |
+| -d   | Calculate the difference to present data.                                           |
+| -s   | Store stats of the difference calculation as the separate file inputmesh.stats.json |
+
+Sample usage is
+
+```bash
+vtk_calculator.py MeshA.vtk -f x+y -t "x + y"
+vtk_calculator.py MappedData.vtk x+y -t difference -it "x + y" --diff
+```
+
+### partition_mesh.py
+
+Reads a mesh either `.vtk` or `.vtu` , partitions it and stores the parts `output_1.vtu, output_2.vtu, ...`. For partitioning, there are two algorithms available. The meshfree algorithm does not need any mesh topology information, whereas the topological algorithm needs topology information. This python module needs the C++ module `libmetisAPI.so` if the topological algorithm is used.
+
+| Flag | Explanation                                                                                 |
+| ---- | ------------------------------------------------------------------------------------------- |
+| -o   | The output mesh name. (default="partitioned_mesh")                                          |
+| -n   | The number of parts to split into                                                           |
+| -a   | Algorithm used for determining the partitioning (options="meshfree", "topology", "uniform") |
+
+Sample usage:
+
+```bash
+partition_mesh.py MeshA.vtk -a topology -n 2 -o fine_mesh
+```
+
+### join_mesh.py
+
+Reads a partitioned mesh from a given prefix (looking for `<prefix>_<#filerank>.vtu)`) and saves it to a single `.vtk` or `.vtu` file.
+Using the `-r` flag, it also recovers the connectivity information from a mesh. Notice that for recovery, partitions should contain `GlobalIDs` data.
+
+| Flag | Explanation                                                                                           |
+| ---- | ----------------------------------------------------------------------------------------------------- |
+| -o   | The output mesh. Can be VTK or VTU format. If it is not given `<inputmesh>_joined.vtk` will be used." |
+| -r   | The path to the recovery file to fully recover it's state.                                            |
+| -n   | The number of parts to read from the input mesh. By default the entire mesh is read.                  |
+| -l   | Logging level (default="INFO")                                                                        |
+
+```bash
+./join_mesh.py partitoned_mesh_directory/partitioned_mesh -r partitioned_directory -o rejoined_mesh.vtk
+```
+
+### libMetisAPI
+
+This is a small C++ wrapper around METIS. It is only required if `partition_mesh.py` should use a topological algorithm.
```

Comment:
Since all previous scripts are files that the user can execute, would maybe this fit better as a short line under `partition_mesh.py`?

Reply:
Definitely I moved to there